tests/testthat/test_get_stan_data.R

```diff
@@ -199,9 +199,21 @@ test_that(paste0(
   "Test that the model handles include_ww = 0 ",
   "and no data appropriately"
 ), {
+  null_ww_data <- NULL
+
+  site_subpop_spine_mod <- get_site_subpop_spine(
+    input_ww_data = null_ww_data,
+    input_count_data = input_count_data
+  )
+
+  lab_site_subpop_spine_mod <- get_lab_site_subpop_spine(
+    lab_site_site_spine = lab_site_site_spine,
+    site_subpop_spine = site_subpop_spine_mod
+  )
+
   stan_data_ho <- get_stan_data(
     input_count_data,
-    input_ww_data = NULL,
+    input_ww_data = null_ww_data,
```

Comment:
Why is this necessary? I think I wanted to actually test that passing in nothing for `input_ww_data` works as expected.

Reply:
just because it's used in two places (also in the computation of `site_subpop_spine_mod`. If you want to test omitting the argument altogether I would delete the named argument rather than passing `NULL`.